features/nanostack/sal-stack-nanostack/nanostack/ws_management_api.h

```diff
@@ -727,6 +727,16 @@ int ws_statistics_start(
     int8_t interface_id,
     ws_statistics_t *stats_ptr);
 
+/**
+ * Reset Wi-SUN statistics.
+ *
+ * \param interface_id Network interface ID.
+ *
+ * \return 0 Success.
+ * \return <0 Failure.
+ */
```

Comment:
Cant make changes to Nanostack and also you should not make changes

Reply:
I agree, usually we have done changes to Nanostack first, then created the Nanostack PR to the Mbed OS. Once the Nanostack PR is merged we can make the actual change to the Mbed OS as the Nanostack API is in available. 
However, this is  rather unpleasant way of working. Could we just make Nanostack changes directly in the Mbed OS PR and then cherry-pick the Nanostack related changes back to the Nanostack?

Reply:
I guess I do not need to make any changes in the sal-stack-nanostack. According to @mikter, the stats structures can be memset to zero from the wisun_tasklet.c file.
I will make this modification.